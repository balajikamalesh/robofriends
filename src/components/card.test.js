import {shallow, render, mount} from 'enzyme';
import React from 'react';
import Card from './card';

//shallow - shallow render (wont render nested components as well)
//mount - full DOM render (will render nested components as well)

//render - rendered to a static HTML (in between shallow and mount)

it('expect to render card component',() => {
    expect(shallow(<Card/>)).toMatchSnapshot();
})