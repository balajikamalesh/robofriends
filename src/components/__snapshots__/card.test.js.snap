// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render card component 1`] = `
<div
  className="bg-light-green dib br3 pa3 ma2 grow"
>
  <img
    alt="robots"
    src="https://robohash.org/undefined?size=200x200"
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
